plugin/src/main/cpp/export/meta_export_plugin.cpp

```diff
@@ -475,6 +497,34 @@ String MetaEditorExportPlugin::_get_android_manifest_application_element_content
 		contents += "        <meta-data android:name=\"com.oculus.ossplash.background\" android:value=\"passthrough-contextual\" />\n";
 	}
 
+	HybridType hybrid_type = _get_hybrid_app_option_value();
+	if (hybrid_type != HYBRID_TYPE_DISABLED) {
+		contents +=
+				"        <activity android:name=\"org.godotengine.openxr.vendors.GodotPanelApp\" "
+				"android:process=\":GodotPanelApp\" "
+				"android:label=\"@string/godot_project_name_string\" "
+				"android:theme=\"@style/GodotAppSplashTheme\" "
```

Comment:
This presumes that this plugin will run with the Godot Android template which is not guaranteed (e.g: developers can integrate this plugin's aar binary into their own project with a different structure), in which case the `GodotAppSlashTheme` will be missing.

To counter that, we should replicate the themes in https://github.com/godotengine/godot/blob/master/platform/android/java/app/res/values/themes.xml in this project (with different names to avoid collision in the common case), and refer to the new themes instead.
Note that we don't need to duplicate the icons specified in the themes as those are provided by the Godot Android library which is a required dependency for this plugin.